src/components/pages/membership_form/subpages/AddressPage.vue

```diff
@@ -1,14 +1,14 @@
 <template>
 	<div class="address-page">
-		<address-fields
-				:validate-address-url="validateAddressUrl"
-				:validate-email-url="validateEmailUrl"
-				:countries="countries"
-				:salutations="salutations"
-				:address-validation-patterns="addressValidationPatterns"
-				:date-of-birth-validation-pattern="dateOfBirthValidationPattern"
-				ref="address">
-		</address-fields>
+		<AddressFields
+			:validate-address-url="validateAddressUrl.toString()"
+			:validate-email-url="validateEmailUrl.toString()"
+			:countries="countries"
+			:salutations="salutations"
+			:address-validation-patterns="addressValidationPatterns"
+			:date-of-birth-validation-pattern="dateOfBirthValidationPattern"
+			ref="addressFieldsRef">
+		</AddressFields>
 		<div class="summary-wrapper has-margin-top-18 has-outside-border">
```

Comment:
There's a new `FormSummary` component for this. Have a look at the Donation Form AddressPage to see an example.